scripts/eosio_build_centos8_deps

```diff
@@ -0,0 +1,28 @@
+git,rpm -qa
+autoconf,rpm -qa
+automake,rpm -qa
+libtool,rpm -qa
+make,rpm -qa
+cmake,rpm -qa
+bzip2,rpm -qa
+doxygen,rpm -qa
+graphviz,rpm -qa
+bzip2-devel,rpm -qa
+openssl-devel,rpm -qa
+gmp-devel,rpm -qa
+ocaml,rpm -qa
+platform-python-devel,rpm -qa
+python36,rpm -qa
+gettext-devel,rpm -qa
+file,rpm -qa
+libusbx-devel,rpm -qa
+libcurl-devel,rpm -qa
+patch,rpm -qa
+llvm-devel,rpm -qa
+llvm-static,rpm -qa
+epel-release,rpm -qa
+jq,rpm -qa
+boost169-devel,rpm -qa
```

Comment:
I don't see any evidence of this boost package being used so it can probably be removed